### PR TITLE
Enable polygon config

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: config/chainbridge-runtime-config.cere.dev.js [hunk 1]
```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
...
+        bridgeAddress: "0x80C8ee1C89c574eB97B3783f0B9C9483EdF8B92E",
+        erc20HandlerAddress: "0x53cc2228587EAca7CcE3189f24BC9bd490E43C48",
+        rpcUrl:
+          "https://polygon-mumbai.infura.io/v3/178b87e860404572bc217ca53fe594bd",
+        type: "Ethereum",
+        nativeTokenSymbol: "MATIC",
+        tokens: [
+          {
+            address: "0xcD54b7066839B25A0E5fDf76866D338cc7257179",
+            name: "CN",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.dev.js]: cereTokenName?

File: config/chainbridge-runtime-config.cere.dev.js [hunk 2]
```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
...
+            symbol: "CS",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.dev.js]: cereTokenSymbol?

**Review by AndreiNavoichyk — APPROVED** [review]

File: config/chainbridge-runtime-config.cere.dev.js (hunk 1 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js]: While deploying the token name is set as 'CN'

File: config/chainbridge-runtime-config.cere.dev.js (hunk 2 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js]: While deploying the token symbol is set as 'CS'

File: config/chainbridge-runtime-config.cere.dev.js (hunk 1 above)
Comment (AndreiNavoichyk) [reply config/chainbridge-runtime-config.cere.dev.js]: Ok, but let's keep it as on PROD for UI.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 2 above)
Comment (AndreiNavoichyk) [reply config/chainbridge-runtime-config.cere.dev.js]: Ok, but let's keep it as on PROD for UI.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 1 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js]: updated code.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 2 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js]: updated code.

**Pull request merged**